modules/con_example-yaml-file-changes.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_example-yaml-file-changes_{context}"]
+= Example YAML file changes
+
+You can dynamically add a code snippet in the YAML editor for customizing a developer catalog.
+
+Use the following snippet to display all the sub-catalogs by setting the _state_ type as *Enabled*.
```

Comment:
Just a suggestion and not required - maybe "set to" rather than "set as" when it is an explicit value -

Reply:
Fixed.